### ISSUE TITLE
Markbot

### ISSUE THREAD

**Issue by anna-pokhilenko** [anna-pokhilenko/just-your-type#1]
Hello) It looks like everything is done, but Markbot doesn't like Thomas' type.css. And I also wanted to make it responsive and added every class needed but it doesn't really work, I have no idea why!

@muxlowc

**Comment by muxlowc** [comment]
Hey @anna-pokhilenko can you send me a screenshot of what Markbot is saying?

**Comment by muxlowc** [comment]
@anna-pokhilenko So the reason your units aren't responding is because you have "rem" units for all of your top/bottom/left/right on your absolute items.. these will always stay at your specified unit, which is why you should try using percentages instead of units such as rem/em/px etc.

Percentages will respond better when you resize the browser - hopefully this helps a bit!

**Comment by anna-pokhilenko** [comment]
@muxlowc 
hey) thanks a lot! that helped a bit and now the part shifts, but I got another issue now.. it overlaps with the "E"
it doesn't want to jump down and take its own space below

**Comment by muxlowc** [comment]
@anna-pokhilenko  If take off the "relative" on your div surrounding your E/every/each etc, and change your "absolute" class to "relative" on your E div, that should work.

`<div class="unit xs-1 s-1 m-1 l-1-2">

      <div class="e relative zetta push bold">
        <h1 class="hidden"></h1>
        <p>E</p>
      </div>`

**Comment by anna-pokhilenko** [comment]
@muxlowc 
done it, but it still doesn't work 😞

**Comment by muxlowc** [comment]
Hey @anna-pokhilenko so sorry, this issue slipped down the list and ended up slipping my mind - I will take a look right now!

**Comment by anna-pokhilenko** [comment]
@muxlowc 
no problem! I am coding right now so perfect timing 😄

**Comment by muxlowc** [comment]
@anna-pokhilenko okay. So. I think I got them to respond better (you'll have to play around a bit to make them centred again) I copied my new code below:

HTML:
`  <div class="grid">

    <div class="unit xs-1 s-1 m-1 l-1-2 block">

      <div class="e relative zetta push bold">
        <h1 class="hidden"></h1>
        <p>E</p>
      </div>

      <div class="enjoy absolute giga">
        <p>njoy</p>
      </div>

      <div class="each absolute giga">
        <p>ach</p>
      </div>

      <div class="every absolute giga">
        <p>very</p>
      </div>

    </div>

    <div class="unit xs-1 s-1 m-1 l-1-2 block first">
      <div class="absolute">
        <div class="">
          <p>moment that comes your way.</p>
        </div>

        <div class="tooltip circle animated infinite pulse">
          <span class="tooltiptext milli">the fire will most likely burn out eventually</span>
        </div>
      </div>
    </div>
  </div>`

CSS:
`
body {
  background-color: #f28729;
}


.enjoy {
  left: 4em;
  top: 5em;
  font-family: "Merriweather Sans", sans-serif;
  color: #f28729;
}

.each {
  top: 32.5rem;
  left: 18.6rem;
  font-family: "Merriweather Sans", sans-serif;
  color: #f28729;
}

.every {
  top: 3rem;
  left: 5em;
  font-family: "Merriweather Sans", sans-serif;
  color: #f28729;
}

.e {
  font-family: Acme, sans-serif;
  top: -7rem;
  left: 4rem;
  color: white;
  margin-bottom: -40%;
}

.first {
  top: 0;
  left: 50%;
  font-family: "Merriweather Sans", sans-serif;
  color: white;
}

.second {
  top: 45rem;
  left: 40rem;
  font-family: "Merriweather Sans", sans-serif;
  color: white;
}

.circle {
  width: 50px;
  height: 50px;
  border-radius: 50%;
  background-color: white;
}

.tooltip {
  position: absolute;
  display: inline-block;
  border-bottom: 1px dotted #a62d85;
}

.tooltip .tooltiptext {
  visibility: hidden;
  left: -6rem;
  font-family: "Merriweather Sans", sans-serif;
  color: #591404;
  width: 300px;
  background-color: white;
  text-align: center;
  padding: 20px 0;
  border-radius: 6px;

  position: absolute;
  z-index: 1;
}

.tooltip:hover .tooltiptext {
  visibility: visible;
}
`

I basically just added in another DIV around your right-hand section which is where absolute works a bit better + I added the class of "Block" to those two sections to force them to not overlap + I moved your "first" class up to it's parent div.
